pinot-core/src/main/java/org/apache/pinot/core/util/CrcUtils.java

```diff
@@ -59,7 +63,8 @@ private static void getAllNormalFiles(File dir, List<File> normalFiles) {
     Preconditions.checkNotNull(files);
     for (File file : files) {
       if (file.isFile()) {
-        if (!file.getName().equals(V1Constants.SEGMENT_CREATION_META)) {
+        // We should ignore both SEGMENT_CREATION_META and generated '.crc' files.
+        if (!file.getName().equals(V1Constants.SEGMENT_CREATION_META) && !file.getName().endsWith(CRC_FILE_EXTENSTION)) {
```

Comment:
Where is the `.crc` file generated?

Reply:
I think I've seen certain data copy operations may leave .crc files in the directory, which could cause the crc check failed.

Reply:
Are you referring to the behavior of some FileSystem? If so, can you please add more comments about that?